lib/vicadmin/validate.go

```diff
@@ -62,6 +63,47 @@ const (
 	BadStatus  = template.HTML(`<i class="icon-attention"></i>`)
 )
 
+func GetMgmtIP() net.IPNet {
+	var mgmtIP net.IPNet
+	// management alias may not be present, try others if not found
+	link := LinkByNameOrAlias("management")
+	if link == nil {
+		link = LinkByNameOrAlias("external")
+	}
+	if link == nil {
+		link = LinkByNameOrAlias("client")
```

Comment:
I think this call is superfluous, since if you don't get the interface from the first call, it pretty much means there is only one interface in the vm, and its being shared by all three networks. So I don't see `link == nil` after the second call above.

Cases:
1. three interfaces: 1st call will succeed, since there is an interface with name `management`
2. two interfaces: 1st call will succeed, since at least one interface has name or alias `management`
3. one interface: 1st call will fail in the case where `client` and `external` are name/alias combo. Second call for either `client` or `external` will work